docs/evaluate/development-production-features/multi-tenant.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: multi-tenant
+title: Multi-tenant - Temporal feature
+description: Learn about Temporal Cloud's multi-tenant architecture and how it enhances scalability, efficiency, and cost-effectiveness.
+sidebar_label: Multi-tenant
+tags:
+- multi-tenant
+
+keywords:
+- multi-tenant
+- Temporal Cloud
+- cloud architecture
+- scalability
+- cost-effectiveness
+- noisy neighbor 
+- database performance
+- high throughput
+---
+
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+Temporal Cloud's architecture scales to handle multiple tenants efficiently.
+Our features enable you to handle multiple Namespaces sharing the same compute and persistence layer, ensuring performance and operational efficiency.
+A Namespace is a unit of isolation within the Temporal Platform.
```

Comment:
I think we can rewrite the rest of this to flow a bit better and emphasize what's good about what we're doing here... here's my attempt:

A Namespace is a unit of isolation within the Temporal Platform -- but even a single Namespace is still multi-tenant. Multi-tenancy ensures extra capacity is available for all customers during traffic spikes.

However, multi-tenancy can also presents the challenge of "noisy neighbors", where high-traffic tenants consume excess resources, causing slower performance for other tenants. This is a common problem for scaling database scaling. 

Temporal's write-heavy workload, where changes in execution state are constantly written to the persistence layer, demands a database that supports reliably high throughput with low latency for multiple customers, concurrently and fairly. With Temporal Cloud, customers pay for consumption instead of entire sets of hardware, providing a cost-effective solution.

Reply:
This is fantastic and definitely agree this flows better! 🚀